userapi/storage/postgres/deltas/20201001204705_last_seen_ts_ip.go

```diff
@@ -5,13 +5,8 @@ import (
 	"fmt"
 
 	"github.com/matrix-org/dendrite/internal/sqlutil"
-	"github.com/pressly/goose"
 )
 
-func LoadFromGoose() {
```

Comment:
This reminds me: we need to just nuke these from all the deltas and remove `/cmd/goose`. In practice, no one uses it and hence we shouldn't carry the baggage for it. The intention was to allow people to rollback DB changes using goose.

Reply:
Agreed.